src/Microsoft.Azure.SignalR.Emulator/Startup.cs

```diff
@@ -39,6 +41,11 @@ public void ConfigureServices(IServiceCollection services)
             {
                 manager.FeatureProviders.Add(new CustomControllerFeatureProvider());
             });
+
+            var upstreamOptions = new UpstreamOptions();
+            Configuration.GetSection("UpstreamSettings").Bind(upstreamOptions);
+            services.Configure<UpstreamOptions>(s => s.Templates = upstreamOptions.Templates);
```

Comment:
Put `Bind` to callback of `Configure<UpstreamOptions>` to support hotreload

Reply:
nice! will address in later pr!